test/built-ins/Object/defineProperties/15.2.3.7-6-a-110.js

```diff
@@ -36,17 +36,19 @@ var properties = {
 };
 
 Object.defineProperties(obj, properties);
-verifyEqualTo(obj, "foo1", 200);
 
-verifyWritable(obj, "foo1");
+verifyProperty(obj, "foo1", {
+  value: 200,
+  writable: true,
+  enumerable: true,
+  configurable: true,
+});
 
-verifyEnumerable(obj, "foo1");
-
-verifyConfigurable(obj, "foo1");
 verifyEqualTo(obj, "foo2", get_func());
```

Comment:
`foo2` is an accessor property, which isn't yet supported by `verifyProperty`. I have some patches to add access properties support to `verifyProperty`, but first I'd like to convert the existing value properties tests to use `verifyProperty` instead of calling `verify{Writable,Enumerable,Configurable}`.

Reply:
aha, ok that makes sense. feel free to ignore all similar suggestions where it's an accessor.